src/test/opentrackIO_parser/python/opentrackIO_parser.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+#
+# opentrackIO_parser.py
+#
+# Reference code for parsing/decoding opentrackIO samples
+#
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the SMTPE RIS OSVP Metadata Project
+#
+# Example run: python3 opentrackIO_parser.py --file=opentrackio_complete_static_example-20241001.json --schema=opentrackio_schema_20241001.json
+# This is tested against the generated "complete_static_example" and "complete_dynamic_example" json
+
+import os
+import argparse
+from opentrackIOlib import *
+
+parser = argparse.ArgumentParser(description='OpenTrackingProtocol parser')
+parser.add_argument('-f', '--file', help='The JSON input file.', default=None)
+parser.add_argument('-s', '--schema', help='The schema (JSON) input file.', default=None)
+parser.add_argument('-v', '--verbose', help='Make script more verbose',action='store_true')
+args = parser.parse_args()
+
+sample_text = ''
+schematext = ''
+verbose = False
+
+if (args.schema):
```

Comment:
What if not all arguments are provided? Could it fail and print usage?

Reply:
Done